sbi/inference/snpe/snpe_c.py

```diff
@@ -38,7 +43,7 @@ def __init__(
 
         Args:
             num_pilot_sims: number of simulations that are run when
-                instantiating an object. Used to z-score the observations.   
+                instantiating an object. Used to z-my_sample_with_mcmc
```

Comment:
something is wrong here, no?

Reply:
I think refactoring in VSCode might be to blame. :rofl:

Reply:
ahahah 100% right Alvaro